docs/pages/api-docs/card-media.md

```diff
@@ -30,7 +30,7 @@ The `MuiCardMedia` name can be used for providing [default props](/customization
 |:-----|:-----|:--------|:------------|
 | <span class="prop-name">children</span> | <span class="prop-type">node</span> |  | The content of the component. |
 | <span class="prop-name">classes</span> | <span class="prop-type">object</span> |  | Override or extend the styles applied to the component. See [CSS API](#css) below for more details. |
-| <span class="prop-name">component</span> | <span class="prop-type">elementType</span> | <span class="prop-default">'div'</span> | Component for rendering image. Either a string to use a HTML element or a component. |
+| <span class="prop-name">component</span> | <span class="prop-type">elementType</span> | <span class="prop-default">'div'</span> | The component used for the root node. Either a string to use a HTML element or a component. |
```

Comment:
Unfortunate but the prop documentation is only useful if you don't know for what component it is. We'll see if up-to-date ts/prop types are worth more than this documentation.